Fix/doc 4319: Kill Shindig documentation (#198)
exoplatform/exo-documentation: docs/Upgrade.rst

your custom extension overrides some Groovy templates, you must check 
if it has been changed, and update it if it is the case.

**Shindig removal**

> WIth eXo Platform 5.3 version, we no more support gadgets as we dropped the use of `Shindig <https://shindig.apache.org/>`__

Reviewer: ```suggestion
In eXo Platform 5.3 version, we no longer support as we dropped `Apache Shindig <https://shindig.apache.org/>` which has been retired.
```
